src/commands/createNewProject.ts

```diff
@@ -0,0 +1,352 @@
+        ];
+
+        // The source images are in the 'images' directory of the extension
+        const sourceImagesPath = path.join(extensionPath, 'images');
+        const targetImagesPath = path.join(projectPath, 'images');
+
+        vscode.window.showInformationMessage(`Copying template images from ${sourceImagesPath} to ${targetImagesPath}`);
+
+        for (const image of imageFiles) {
+            try {
+                const sourcePath = path.join(sourceImagesPath, image.src);
+                const targetPath = path.join(targetImagesPath, image.dest);
+                
+                // Check if target image already exists
+                if (await pathExists(targetPath)) {
+                    const overwrite = await vscode.window.showWarningMessage(
+                        `Image "${image.dest}" already exists. Do you want to overwrite it?`,
+                        'Yes', 'No'
+                    );
+                    if (overwrite !== 'Yes') {
+                        continue;
+                    }
+                }
+                
+                vscode.window.showInformationMessage(`Copying ${sourcePath} to ${targetPath}`);
+                await fs.promises.copyFile(sourcePath, targetPath);
+            } catch (error) {
+                vscode.window.showWarningMessage(`Warning: Failed to copy template image ${image.src}: ${error.message}`);
+            }
+        }
+    } catch (error) {
+        throw new Error(`Failed to copy template images: ${error.message}`);
+    }
+}
+
+/**
+ * Creates launch.json file for debugging configuration
+ */
+async function createLaunchJson(projectPath: string) {
+    const vscodePath = path.join(projectPath, '.vscode');
+    await fs.promises.mkdir(vscodePath, { recursive: true });
+
+    const launchConfig = {
+        version: '0.2.0',
+        configurations: [
+            {
+                name: 'BrightScript Debug: Launch',
+                type: 'brightscript',
+                request: 'launch',
+                host: '${promptForHost}',
+                password: '${promptForPassword}',
+                rootDir: '${workspaceFolder}',
+                enableDebuggerAutoRecovery: false,
+                stopOnEntry: false,
+                enableDebugProtocol: true
+            }
+        ]
+    };
+
+    const launchPath = path.join(vscodePath, 'launch.json');
+    
+    // Check if launch.json already exists
+    if (await pathExists(launchPath)) {
+        const overwrite = await vscode.window.showWarningMessage(
+            'launch.json already exists. Do you want to overwrite it?',
+            'Yes', 'No'
+        );
+        if (overwrite !== 'Yes') {
+            return;
+        }
+    }
+
+    await fs.promises.writeFile(
+        launchPath,
+        JSON.stringify(launchConfig, null, 4)
+    );
+}
+
+/**
+ * Creates all necessary project files with template content
+ */
+async function createProjectFiles(projectPath: string, projectName: string) {
```

Comment:
Let's move the Roku project files out of typescript strings and instead store them as actual files in the project. Maybe call the folder `project-templates`. Then you can do a simple `fastGlob` lookup to find all files in there recursively and copy them to the `projectPath`.

Then for inserting the `projectName` and other variables, let's just establish some standard names that we can do a quick replacement on as we copy them. (like `title=VscodeTemplateProjectName`)

This has a few benefits:
 - it's easier to scale out more starter projects 
 - the projects will actually be runnable on their own from their folders
 - it's easier to maintain compared to strings